src/psyclone/tests/alggen_test.py

```diff
@@ -391,7 +416,7 @@ def test_single_invoke_dynamo0p1(self):
                          "test_files", "dynamo0p1", "algorithm",
                          "1_single_function.f90"), api="dynamo0.1")
         gen = str(alg)
-        assert "USE single_function_psy, ONLY: invoke_testkern_type" in gen
+        assert "USE psy_single_function, ONLY: invoke_0_testkern_type" in gen
         assert "CALL invoke_0_testkern_type(f1, f2, m1)" in gen
 
     def test_zero_invoke_dynamo0p1(self):
```

Comment:
There a `pylint` warning (R:422, 4: Method could be a function (no-self-use)`). Please see the remark above about possible re-purposing this test as Dynamo0.1 API is to be deleted.

Reply:
Fixed.